### PR TITLE
Redis use master

### LINKED ISSUE #1292: 统一redis 使用为 哨兵模式

### PR DESCRIPTION
**Pull request by holmeswang**
### 优化的功能:
- webserver 支持redis 哨兵模式 close #1292

### CODE REVIEW

File: src/web_server/service/service.go [hunk 1]
```diff
@@ -40,9 +41,20 @@ type Service struct {
...
+		address := s.Config.Session.Host + ":" + s.Config.Session.Port
+		store, redisErr = sessions.NewRedisStore(10, "tcp", address, s.Config.Session.Secret, []byte("secret"))
+		if redisErr != nil {
+			panic(redisErr)
+		}
+	} else {
+		address := strings.Split(s.Config.Session.Address, ";")
+		store, redisErr = sessions.NewRedisStoreWithSentinel(address, 10, s.Config.Session.MasterName, "tcp", s.Config.Session.Secret, []byte("secret"))
+		if redisErr != nil {
+			panic(redisErr)
```
Comment (breezelxp) [inline src/web_server/service/service.go]: 改成blog.Fatal()

File: src/web_server/service/service.go [hunk 2]
```diff
@@ -22,9 +22,10 @@ import (
 	"configcenter/src/web_server/app/options"
 	"configcenter/src/web_server/logics"
 	"configcenter/src/web_server/middleware"
+	"strings"
```
Comment (breezelxp) [inline src/web_server/service/service.go]: ?

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=1305) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=1305) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: holmeswang<br/>:white_check_mark: breezelxp<br/>:x: lampardtang<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-cmdb?pullRequest=1305) it.</sub>

File: src/web_server/service/service.go (hunk 2 above)
Comment (holmeswang) [reply src/web_server/service/service.go]: 已修改

File: src/web_server/service/service.go (hunk 1 above)
Comment (holmeswang) [reply src/web_server/service/service.go]: 已修改

**Pull request merged**